server/src/db/seeders/UserSeeder.js

```diff
@@ -4,14 +4,17 @@ class UserSeeder {
     static async seed() {
         const usersData = [
             {
+                username: "JohnDoe22",
                 email: "johnDoe22@email.com",
                 cryptedPassword: "12345"
```

Comment:
if you want to be able to log in as these seeded users, you will need to create them with a `password` instead of `cryptedPassword` - the User model will handle encrypting the passwords for us in the database